setup.py

```diff
@@ -45,9 +44,10 @@
         'plone.app.registry',
         'plone.registry',
         'Products.CMFCore',
-        'Products.CMFPlone >=4.2',
+        'Products.CMFPlone >=4.3',
```

Comment:
Pinning PlonePAS/PluggableAuthService seems sufficient. Then someone can choose to use say CMFPlone 4.3.0 and change the default pins for the PAS packages.